chromium_src/components/variations/variations_url_constants.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/variations/variations_url_constants.h"
+
+namespace variations {
+
+// Default server of Variations seed info.
+const char kDefaultServerUrl[] =
+    "https://variations.brave.com/seed";
```

Comment:
depending on when this is fetched, you may need to add it to https://github.com/brave/brave-core/blob/master/build/commands/lib/whitelistedUrlPrefixes.js so that the network audit CI check passes

Reply:
@diracdeltas this is fetched on startup. Do I need to create a ticket for sec review to add to the list?

Reply:
@moritzhaller you can just add `https://variations.brave.com/seed` to the list. thanks!

Reply:
great, added!